### ISSUE TITLE
Design Intent - Accessibility Feedback - Veteran Facing Forms - Form Review page changes (VADS pattern)

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#85989]
## Next Steps for the VFS team

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

- Very excited about this!

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] **Governance note:** Just noting that if adopting the Edit by Page model, forms using this review page will need to have a unique heading for each page --- which is not currently true for all forms. Some forms have no heading, some have multiple headings, some have a `legend`/`fieldset` but the `legend` isn't coded as a heading, etc. No action required by you, except that when it comes time to update VADS guidance/forms library we'll need to account for that.
- [ ] **Should:** I definitely prefer the continuous error state flow over the hub and spoke model. Some thoughts justifying that:
   - Multiple error alerts (with `role="error"`) plus focus going to the success alert could result in some undesirable screen reader behavior. I think there's a chance the errors will conflict with each other on some screen readers, and I'm pretty sure (but have not tested) that the errors will probably be announced before the success message, so users would hear the "you need to fix this" then "success, you did it" --- which could be confusing.
   - Users going back and making changes haven't done anything *wrong*, and so they shouldn't get messages that indicate an error.
   - The interstitial "hey since you did X now we need to know more stuff" page ensures that users are aware of what happened, no chance of it being missed somehow.
- [ ] **Must:** If you go with the continuous flow, make sure you account for the use case where a user makes a change, leaves the form, and comes back. I kind of don't want them to go straight to the review page --- maybe straight to the interstitial page? Something to explore.
- [ ] **Consider:** In the continuous flow, consider adding some kind of message to the interstitial page telling users how many fields they need to provide responses for now, if it's possible to do so programmatically.
- [ ] **Consider:** Consider adding content to the interstitial page reminding authenticated users that their work is being saved and they can bail when they need to and then come back. Thinking of the cognitive load/anxiety concerns around "I fixed one little thing and made an extra 30 minutes of work for myself."

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [x] Assign this ticket to the VFS team member that opened the Slack request
- [x] Add the VFS team product label
- [x] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [x] Add the Collaboration Cycle initiative milestone

**Comment by jeana-adhoc** [comment]
All of these items have been considered and documented in the [design decisions page](https://www.figma.com/design/kBH2U6bfC6sA9Fim3gcnuc/WIP---Review-Page---Edit-by-page?node-id=432-65242&node-type=INSTANCE&t=i9kQjfr194H3T4Xz-11) of the figma file.
